[BNO055 @ factory.py:85][DEBUG] 2017-10-28 06:51:01,577: Logging to: logs/2017_Oct_28/BNO055/06_51_01.log
[BNO055 @ arduino.py:76][INFO] 2017-10-28 06:51:01,617: configuring for the first time. whoiam=BNO055-IMU
[BNO055 @ arduino.py:79][INFO] 2017-10-28 06:51:01,629: Found suitable addresses: '['/dev/ttyUSB2', '/dev/ttyUSB1', '/dev/ttyUSB0']'
[BNO055 @ arduino.py:285][DEBUG] 2017-10-28 06:51:01,631: /dev/ttyUSB2 is ready
[BNO055 @ arduino.py:285][DEBUG] 2017-10-28 06:51:01,631: /dev/ttyUSB1 is ready
[BNO055 @ arduino.py:285][DEBUG] 2017-10-28 06:51:01,632: /dev/ttyUSB0 is ready
[BNO055 @ arduino.py:354][DEBUG] 2017-10-28 06:51:03,223: Checking 'whoareyou' protocol
[BNO055 @ arduino.py:354][DEBUG] 2017-10-28 06:51:03,224: Checking 'whoareyou' protocol
[BNO055 @ arduino.py:354][DEBUG] 2017-10-28 06:51:03,225: Checking 'whoareyou' protocol
[BNO055 @ arduino.py:388][DEBUG] 2017-10-28 06:51:03,432: received packet: 'iamBNO055-IMU'
[BNO055 @ arduino.py:317][DEBUG] 2017-10-28 06:51:03,435: Found ID 'BNO055-IMU'
[BNO055 @ arduino.py:354][DEBUG] 2017-10-28 06:51:03,435: Checking 'init?' protocol
[BNO055 @ arduino.py:388][DEBUG] 2017-10-28 06:51:03,624: received packet: 'init:'
[BNO055 @ arduino.py:339][DEBUG] 2017-10-28 06:51:03,640: sent initialization data: ''
[BNO055 @ arduino.py:136][INFO] 2017-10-28 06:51:03,656: address '/dev/ttyUSB2' has ID 'BNO055-IMU'
[BNO055 @ arduino.py:398][DEBUG] 2017-10-28 06:51:04,488: Writing 'whoareyou' again
[BNO055 @ arduino.py:398][DEBUG] 2017-10-28 06:51:04,548: Writing 'whoareyou' again
[BNO055 @ arduino.py:398][DEBUG] 2017-10-28 06:51:05,056: Writing 'whoareyou' again
[BNO055 @ arduino.py:398][DEBUG] 2017-10-28 06:51:05,168: Writing 'whoareyou' again
[BNO055 @ arduino.py:398][DEBUG] 2017-10-28 06:51:05,660: Writing 'whoareyou' again
[BNO055 @ arduino.py:398][DEBUG] 2017-10-28 06:51:05,768: Writing 'whoareyou' again
[BNO055 @ arduino.py:121][WARNING] 2017-10-28 06:51:06,256: Didn't receive response for packet 'whoareyou' on address '/dev/ttyUSB1'. Operation timed out.
[BNO055 @ arduino.py:398][DEBUG] 2017-10-28 06:51:06,348: Writing 'whoareyou' again
[BNO055 @ arduino.py:121][WARNING] 2017-10-28 06:51:06,348: Didn't receive response for packet 'whoareyou' on address '/dev/ttyUSB0'. Operation timed out.
[BNO055 @ arduino.py:86][INFO] 2017-10-28 06:51:06,453: configuring done. Finding whoiam=BNO055-IMU
[BNO055 @ arduino.py:157][INFO] 2017-10-28 06:51:06,482: Device has started!
[BNO055 @ arduino.py:158][INFO] 2017-10-28 06:51:06,482: init packet: 
[BNO055 @ node.py:102][INFO] 2017-10-28 06:51:08,246: Node took 4.869978189468384s to run
[BNO055 @ generic.py:64][INFO] 2017-10-28 06:51:08,250: Tearing down device
